src/Microsoft.Health.Dicom.Api/Controllers/StoreController.cs

```diff
@@ -40,6 +41,8 @@ public StoreController(IMediator mediator, ILogger<StoreController> logger)
         }
 
         [AcceptContentFilter(new[] { KnownContentTypes.ApplicationDicomJson }, allowSingle: true, allowMultiple: false)]
+        [Produces(KnownContentTypes.ApplicationDicomJson)]
+        [Consumes(KnownContentTypes.ApplicationDicom, new[] { KnownContentTypes.MultipartRelated + TypeSeperator + KnownContentTypes.ApplicationDicom })]
```

Comment:
nit: the second parameter is `params`, so the allocation of an array should be unnecessary (as it will happen behind the scenes).

Also, this `public ProducesAttribute (string contentType, params string[] additionalContentTypes)` ctor reminds me of lisp lol

Reply:
Fixed